openpype/tools/settings/settings/widgets.py

```diff
@@ -730,17 +797,37 @@ def refresh(self):
                 self.dbcon = None
                 self.current_project = None
 
+        items = [(self.default, True)]
+
         if self.dbcon:
-            database = self.dbcon.database
-            for project_name in database.collection_names():
-                project_doc = database[project_name].find_one(
-                    {"type": "project"},
-                    {"name": 1}
+
+            for doc in self.dbcon.projects(
+                    projection={"name": 1, "data.active": 1},
+                    only_active=self._only_active
```

Comment:
This should be always `False`. You want have them in project model all the time but filter them with proxy model.

Reply:
Oh it;s because of SyncServer?

Reply:
Ouch, this currently still being shared with the Sync-Server. Have thought about letting Sync-Server to have it's own project list like you suggested in chat, but that requires a bit more code change to do so. So as current workaround, I make that as optional and fix to `False` once Sync-Server has it's own widget. Maybe in another PR ?

Reply:
> Oh it;s because of SyncServer?

XD, didn't see the update. Yes :D

Reply:
Ah, forgot to mention. Local Settings' project list widget also inherited this.